.github/workflows/npm-release-reusable.yml

```diff
@@ -62,7 +62,9 @@ jobs:
         run: |
           yarn config set version-git-tag false
           # run the bump twice to remove -alpha first
-          yarn version --${{ needs.prepare.outputs.bump }}
+          if [[ $(cat package.json | jq -r .version) == *"-alpha" ]]; then
+            yarn version --${{ needs.prepare.outputs.bump }}
+          fi
```

Comment:
Does this version bump affect the contracts only? Or also the bindings for the Ethers NPM package?

Reply:
the packages `@aragon/core-contracts` and `@aragon/core-contracts-ethers`